# RADseq data assembly for branching _Porites_ spp.

This is an assembly of ddRADseq and EpiRADseq data for branching _Porites_ spp. corals from Belize. Data files constitute 100bp paired-end reads on the Illumina HiSeq4000. 96 samples were prepared on a 96 well plate and can be identified by their unique combination of indices and barcodes. The 12 indices were grouped into 12 separate libraries that were submitted for sequencing. They were combined in equimolar ratios on 1 HiSeq4000 lane.

Further information on these data can be found here: https://github.com/jldimond/jldimond-fish546-2016/tree/master/data/readme.md

In [1]:
!pwd

/Volumes/toaster/jdimond/Branching-Porites/notebooks


In [15]:
cd ../data/Raw_data_JD001

/Volumes/toaster/jdimond/Branching-Porites/data/Raw_data_JD001


In [ ]:
#raw data files must be downloaded (approx. 32GB)
!wget -r --no-parent http://owl.fish.washington.edu/nightingales/Porites_spp/

In [3]:
!ls

JD001_A_S21_L004_R1_001.fastq.gz     JD001_G_S27_L004_R2_001.fastq.gz
JD001_A_S21_L004_R2_001.fastq.gz     JD001_H_S28_L004_R1_001.fastq.gz
JD001_B_S22_L004_R1_001.fastq.gz     JD001_H_S28_L004_R2_001.fastq.gz
JD001_B_S22_L004_R2_001.fastq.gz     JD001_I_S29_L004_R1_001.fastq.gz
JD001_C_S23_L004_R1_001.fastq.gz     JD001_I_S29_L004_R2_001.fastq.gz
JD001_C_S23_L004_R2_001.fastq.gz     JD001_J_S30_L004_R1_001.fastq.gz
JD001_D_S24_L004_R1_001.fastq.gz     JD001_J_S30_L004_R2_001.fastq.gz
JD001_D_S24_L004_R2_001.fastq.gz     JD001_K_S31_L004_R1_001.fastq.gz
JD001_E_S25_L004_R1_001.fastq.gz     JD001_K_S31_L004_R2_001.fastq.gz
JD001_E_S25_L004_R2_001.fastq.gz     JD001_L_S32_L004_R1_001.fastq.gz
JD001_F_S26_L004_R1_001.fastq.gz     JD001_L_S32_L004_R2_001.fastq.gz
JD001_F_S26_L004_R2_001.fastq.gz     Undetermined_S0_L004_R1_001.fastq.gz
JD001_G_S27_L004_R1_001.fastq.gz     Undetermined_S0_L004_R2_001.fastq.gz


In [4]:
## a peak at at read 1 file
!gunzip -c JD001_A_S21_L004_R1_001.fastq.gz | head -n 12

@K00188:197:HFL2KBBXX:4:1101:1570:1508 1:N:0:NTCACG
TGCATTGCCGTGGTTTTTCACTTCGCGGCCGTCCAGTCGAGCTGTGGAATGAAATGTCCTGGCAGGCGGGTGATAGCATGCATACTTAATAATACGCGAG
+
A-AF<<<---7AAAAFJJF<F-FFA<7-7F-FF<JJJFFJFFJJ-7A7---<7F--<--<-A-FJF---7--F7JJ7AAF-<-FAFJJJFFJJJ<<<-A7
@K00188:197:HFL2KBBXX:4:1101:2828:1508 1:N:0:NTCACG
TGCATCGCACTCGAGCTACAGTCGAGCTAGCTTAGTACGGCAGTAGCACGACGTTTGAAACAGGACTAATGTTAGTAAACTGAAATTTTTTAAAGGAGCG
+
A<AFFJ--77AA<J<F<FFJJFJAFJJJFFFJFJ<FFFFFFJFJJJJFJAFJ<A<J<FJFJ<---AFAJ<A-JJJ-FJJJ-FJJ7<F-7A7F-FJJF-77
@K00188:197:HFL2KBBXX:4:1101:3031:1508 1:N:0:NTCACG
TGCATTGCAGCGCGTTGTCGAACGGCAGGGACATCCACGGGCAAGGTGTGGACTTGCGCTTGACCTTCTCGGGGCGTTAAATGACTCCACAATCGGTAGC
+
AAFFF-7<-<77F<JJJFJJAJJJJJ<FFJJJ<<AAFJ<-<<JFFJFJJJAJJ7-FF7<FJJJF-AJFAFJJAJJ7-FFAJJJA7-FAJAJFFJAFFFJJ
gunzip: error writing to output: Broken pipe
gunzip: JD001_A_S21_L004_R1_001.fastq.gz: uncompress failed


In [5]:
## Now let's look at read 2
!gunzip -c JD001_A_S21_L004_R2_001.fastq.gz | head -n 30

@K00188:197:HFL2KBBXX:4:1101:1570:1508 2:N:0:NTCACG
NGCGGATTTGTTAAGCCTGCGCACGTTTTATAAATTTTACGTACACTTAAGTTTTCGCACATTTTATTAAGTTTGCATGCGTTCTGTTTAGAAGTTTGCA
+
#---<77A-FJ-FF7AF7--7FJJJ7-7FFFFJJJ-FFAJFJFJJFJJ<--JA<JJJ<JJJ7<F<F7FF<FFJA7AA-7-7-<7-A--FJJ--FF-----
@K00188:197:HFL2KBBXX:4:1101:2828:1508 2:N:0:NTCACG
NCCATTCAAGAATCTGGATTACCTAACATAAAGCGATACCTTAGTTGCTTACTAACATTTAAAACCACCCACCATCACAGTCATTTCCATTCTTTACGAA
+
#-<7<-FJJJAAFJ7JAJJ<FJJJFJJJJJJFJJFJFJJJF-FJAFFJFAJJ<JFJJFFAJJAAJJJJFJFJJFFJFJJJ-FF-FAFJFFFF--7-A7FF
@K00188:197:HFL2KBBXX:4:1101:3031:1508 2:N:0:NTCACG
NGGACAAATGCGTCTTGGATTTGTTCACCAAAGATTCCCATCATCGCAACATCACCGTGCCCTATTTGACGCAACATTTATTCCCGCCTGGGAAATTCTC
+
#7<<FFJA-FJF-F-<7F7-77<FAJ<JJJJJJJJJJJJJJFAJJJJJJJJ-FFJFJ-FJ--AFF-JA<FFFJJJFFJFJJFJJJAF7-A<JAJJ<-7FJ
@K00188:197:HFL2KBBXX:4:1101:4411:1508 2:N:0:NTCACG
NGGAAGATAAACCATAGGCTCCGTCTCACAACCCTTGCACACACAGCTGTAAAGTGGTAAAGAGTATGGAACGTAGTCCCCCCGTTGTGGTGGTCGATCT
+
#<-<FFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJFJJJJAJJFJJJJJJJJJJJJJJJFJJF

In [6]:
!gunzip -c JD001_A_S21_L004_R1_001.fastq.gz | wc -l

 88358900


In [7]:
#Reads 1 and 2 should have identical numbers of records
!gunzip -c JD001_A_S21_L004_R2_001.fastq.gz | wc -l

 88358900


In [18]:
cd ..

/Volumes/toaster/jdimond/Branching-Porites/data


In [19]:
##Barcodes file for library A.
!cat barcodes_A.txt

80_epi	TGCAT
80_ddr	AAGGA
116_epi	GCATG
108_epi	AACCA
98_ddr	CAACC
82_epi	TCGAT
116_ddr	CGATC
102_epi	GGTTG

# Create a _Symbiodinium_ genome database to filter out symbiont reads

The _Symbiodinium microadriaticum_ (clade A, most ancestral lineage) genome from:

Aranda, M., Li, Y., Liew, Y.J., Baumgarten, S., Simakov, O., Wilson, M.C., Piel, J., Ashoor, H., Bougouffa, S., Bajic, V.B. and Ryu, T., 2016. Genomes of coral dinoflagellate symbionts highlight evolutionary adaptations conducive to a symbiotic lifestyle. Scientific Reports, 6.

In [21]:
cd ./Sym_genome

/Volumes/toaster/jdimond/Branching-Porites/data/Sym_genome


In [22]:
!wget ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/939/145/GCA_001939145.1_ASM193914v1/GCA_001939145.1_ASM193914v1_genomic.fna.gz

--2017-02-13 09:21:04--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/939/145/GCA_001939145.1_ASM193914v1/GCA_001939145.1_ASM193914v1_genomic.fna.gz
           => 'GCA_001939145.1_ASM193914v1_genomic.fna.gz'
Resolving ftp.ncbi.nlm.nih.gov... 130.14.250.10, 2607:f220:41e:250::10
Connecting to ftp.ncbi.nlm.nih.gov|130.14.250.10|:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... done.    ==> PWD ... done.
==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/001/939/145/GCA_001939145.1_ASM193914v1 ... done.
==> SIZE GCA_001939145.1_ASM193914v1_genomic.fna.gz ... 233167110
==> PASV ... done.    ==> RETR GCA_001939145.1_ASM193914v1_genomic.fna.gz ... done.
Length: 233167110 (222M) (unauthoritative)

GCA_001939145.1_ASM 100%[=====================>] 222.37M  5.92MB/s   in 36s    

2017-02-13 09:21:41 (6.20 MB/s) - 'GCA_001939145.1_ASM193914v1_genomic.fna.gz' saved [233167110]



In [23]:
!gunzip -c GCA_001939145.1_ASM193914v1_genomic.fna.gz | head -n 30

>LSRX01000001.1 Symbiodinium microadriaticum strain CCMP2467 Smic.scaffold1, whole genome shotgun sequence
tacaagagtgctgcctgctggcgagagagtccactattgagaagcttcgaagtaaagaattctgagctatggctgcaaag
agccccagcaagtgccgtgtgtggggagctgggactaccctccctagagagagagcGCCACATCAACTGCCAGGTTCCCA
CAATCTCCCTGTGCATGGTACTGTGCACCTGTAATCCAGAGAAGAAATTGCGGAAGTTCAGGTGGCATCATCGCGTCTTG
CATGCCAAGGTTTATCTTGCGTTGGGACCAGTCAATTGGCAGTGCGACTGcgaatcctccaaaccctgtcgTTGGTGGTC
CGCTCGCCGCTGATTTATGTTTTGGGTCCTGCGTTTGACACAAGATCCGAACTGCATAGCAAGTCTAAATGCCTGGACAT
AGCTGAGATCTCGTCGCCAGATCTCcgacagaccttgccaggcttggttAACATGACTGGACGTCAACTTGTTCAGTTTG
AGTCTTGATGTGTTGCTACAGCTCCGGCCTGTGGATACGAGGTGCACTGCTTGATTTGAGCGCATGAAGAGGAAGTAAAT
GGCGAGAGCCTCGACTacgaagtttagtgtttagcatcATGAAAAGTGTTTGCAGCGACCAGTCCGTGTAGTCGGAGAAT
GTCTGAAAGTTCATGCTCGTCCCTCGCATTCCGATATTTCATGTTCATGACTTCTGTATCGGTCACTCAGCATGCGTAAC
TACGTATTGAAATCGACACAGCAGCTCCGACGAGTGAGAGGTGGCTTCACGAGTGATAAACCAAGTTGTTTGCTGACTAA
TCTTGGTTTCGAAAAGGCACTTATCCCCAACAAAATCCACCCGTTTTTTTTTTACTCTTCAGATTTCTCTTCAGATCCAA
TA

In [9]:
!gunzip /Volumes/toaster/jdimond/Branching-Porites/data/Sym_genome/GCA_001939145.1_ASM193914v1_genomic.fna.gz

# Now the raw RAD reads must be demultiplexed

## Create an ipyrad params file for each library with the correct path to the barcodes file


In [2]:
cd ../../analyses/ipyrad_analysis

/Volumes/toaster/jdimond/Branching-Porites/analyses/ipyrad_analysis


In [26]:
#ipyrad uses a simple text file to hold all the parameters for a given assembly. 
#Start by creating a new params file using the -n flag, followed by a name for your assembly.
#Start with a file for library JD001_A
!ipyrad -n data1_A


    New file 'params-data1_A.txt' created in /Volumes/toaster/jdimond/Branching-Porites/analyses/ipyrad_analysis



This will create a file in the current directory called params-data1.txt. The params file lists on each line one parameter followed by a ## mark, then the name of the parameter, and then a short description of its purpose. Edit the file in a text editor.

In [30]:
!cat params-data1_A.txt

------- ipyrad params file (v.0.5.15)-------------------------------------------
data1_A                        ## [0] [assembly_name]: Assembly name. Used to name output directories for assembly steps
./                             ## [1] [project_dir]: Project dir (made in curdir if not present)
../../data/Raw_data_JD001/JD001_A*.fastq.gz                               ## [2] [raw_fastq_path]: Location of raw non-demultiplexed fastq files
../../data/barcodes_A.txt                        ## [3] [barcodes_path]: Location of barcodes file
                               ## [4] [sorted_fastq_path]: Location of demultiplexed/sorted fastq files
denovo-reference                         ## [5] [assembly_method]: Assembly method (denovo, reference, denovo+reference, denovo-reference)
../../data/Sym_genome/GCA_001939145.1_ASM193914v1_genomic.fna.gz                               ## [6] [reference_sequence]: Location of reference sequence file
pairddrad                           ## [7] [datatype]:

For the demultiplexing step, will need params file for each separate library A-L. Edit the assembly name, raw fastq path, barcode path, and filename in a text editor to create params files A-L. 

## Run step 1 for each of the 12 JD001 libraries to demultiplex the data

In [33]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_A.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_A
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:10:27 
  [####################] 100%  sorting reads         | 0:07:02 
  [####################] 100%  writing/compressing   | 2:31:07 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [34]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_B.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  loading Assembly: data1_B
  from saved path: /Volumes/toaster/jdimond/ipyrad_analysis/data1_B.json
  New Assembly: data1_B
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:13:02 
  [####################] 100%  sorting reads         | 0:09:17 
  [####################] 100%  writing/compressing   | 0:43:06 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [35]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_C.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_C
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:10:34 
  [####################] 100%  sorting reads         | 0:07:04 
  [####################] 100%  writing/compressing   | 1:10:28 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [36]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_D.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_D
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:11:03 
  [####################] 100%  sorting reads         | 0:07:23 
  [####################] 100%  writing/compressing   | 0:47:02 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [37]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_E.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_E
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:11:14 
  [####################] 100%  sorting reads         | 0:07:06 
  [####################] 100%  writing/compressing   | 0:39:23 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [38]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_F.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_F
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:11:30 
  [####################] 100%  sorting reads         | 0:08:00 
  [####################] 100%  writing/compressing   | 0:57:44 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [39]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_G.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_G
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:10:29 
  [####################] 100%  sorting reads         | 0:07:16 
  [####################] 100%  writing/compressing   | 1:10:25 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [40]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_H.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_H
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:10:24 
  [####################] 100%  sorting reads         | 0:06:55 
  [####################] 100%  writing/compressing   | 0:45:20 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [41]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_I.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_I
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:09:42 
  [####################] 100%  sorting reads         | 0:05:46 
  [####################] 100%  writing/compressing   | 0:54:53 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [42]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_J.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_J
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:11:10 
  [####################] 100%  sorting reads         | 0:06:54 
  [####################] 100%  writing/compressing   | 1:10:59 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [43]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_K.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_K
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:09:49 
  [####################] 100%  sorting reads         | 0:06:26 
  [####################] 100%  writing/compressing   | 0:54:45 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


In [44]:
## -p indicates the params file we wish to use
## -s indicates the step to run (in this case 1)
!ipyrad -p params-data1_L.txt -s 1


 -------------------------------------------------------------
  ipyrad [v.0.3.41]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1_L
  local compute node: [16 cores] on hummingbird.local

  Step 1: Demultiplexing fastq data to Samples
  [####################] 100%  chunking large files  | 0:09:30 
  [####################] 100%  sorting reads         | 0:06:01 
  [####################] 100%  writing/compressing   | 1:07:30 

  Step 3: Clustering/Mapping reads

    Assembly method reference requires that you enter a 'reference_sequence_path'.
    


## Now combine demultiplexed sample fastqs into a single directory and run step 2 to filter the reads 

For the entire analysis to work properly it needs it to start with all samples in a single directory. FASTQs were manually relocated to ./analyses/ipyrad_analysis/data1_fastqs. Some of these FASTQs (any with sample number under 100 or starting with the letter "w") were removed because they are not branching Porites spp. and are for separate studies. Also, samples 102 and 120 were compromised during library prep and were removed.

Edit a params file with the path to the compiled fastqs (under sorted fastq path), name it "params-data1_all.txt", and run step 2 (must run 1 and 2).

In [7]:
!cat params-data1.txt

------- ipyrad params file (v.0.5.15)-------------------------------------------
data1                        ## [0] [assembly_name]: Assembly name. Used to name output directories for assembly steps
./                             ## [1] [project_dir]: Project dir (made in curdir if not present)
                               ## [2] [raw_fastq_path]: Location of raw non-demultiplexed fastq files
                               ## [3] [barcodes_path]: Location of barcodes file
./data1_fastqs/*.fastq.gz            ## [4] [sorted_fastq_path]: Location of demultiplexed/sorted fastq files
denovo-reference                         ## [5] [assembly_method]: Assembly method (denovo, reference, denovo+reference, denovo-reference)
../../data/Sym_genome/GCA_001939145.1_ASM193914v1_genomic.fna.gz        ## [6] [reference_sequence]: Location of reference sequence file
pairddrad                           ## [7] [datatype]: Datatype (see docs): rad, gbs, ddrad, etc.
TGCAG, NGG                         #

In [8]:
!ipyrad -p params-data1.txt -s 1234567


 -------------------------------------------------------------
  ipyrad [v.0.5.15]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  New Assembly: data1
  local compute node: [16 cores] on hummingbird.fish.washington.edu

  Step 1: Loading sorted fastq data to Samples
  [####################] 100%  loading reads         | 0:03:34  
  112 fastq files loaded to 56 Samples.

  Step 2: Filtering reads 
  [####################] 100%  processing reads      | 0:52:29  

  Step 3: Clustering/Mapping reads
    *************************************************************
    Indexing reference sequence with bwa. 
    This only needs to be done once, and takes just a few minutes
    ************************************************************* 
ERROR:ipyrad.core.assembly:IPyradWarningExit: 
    Reference sequence must be uncompressed fasta or bgzip compressed,
    your file is probably gzip compressed. The simplest fix is to gun

In [10]:
!ipyrad -p params-data1.txt -f -s 34567


 -------------------------------------------------------------
  ipyrad [v.0.5.15]
  Interactive assembly and analysis of RAD-seq data
 -------------------------------------------------------------
  loading Assembly: data1
  from saved path: /Volumes/toaster/jdimond/Branching-Porites/analyses/ipyrad_analysis/data1.json
  local compute node: [16 cores] on hummingbird.fish.washington.edu

  Step 3: Clustering/Mapping reads
    *************************************************************
    Indexing reference sequence with bwa. 
    This only needs to be done once, and takes just a few minutes
    ************************************************************* 
    Done indexing reference sequence
  [####################] 100%  dereplicating         | 1:18:56  
  [####################] 100%  mapping               | 4:52:05  
  [####################] 100%  clustering            | 0:19:19  
  [####################] 100%  building clusters     | 0:03:26  
  [####################] 100%  chu